cmd/minishift/cmd/start.go

```diff
@@ -243,6 +244,30 @@ func applyAddOns(ip string, routingSuffix string, ocPath string, kubeConfigPath
 	}
 }
 
+// TODO - persistent volume creation should really be fixed upstream, aka 'cluster up' (HF)
+// configurePeristentVolumes makes sure that the default persistent volumes created by 'cluster up' have the right permissions - see https://github.com/minishift/minishift/issues/856
+func configurePeristentVolumes(sshCommander provision.SSHCommander) {
+	fmt.Println("-- Waiting for persistent volumes to be ready")
+
+	hostPvDir := viper.GetString(hostPvDir)
+
+	// there are in total 101 directories when using ls. There is an additonal 'registry' directory
+	cmd := fmt.Sprintf("while [ $(ls %s | wc -l) -lt 101 ] ; do sleep 1; done", hostPvDir)
```

Comment:
@hferentschik not at the moment. I will need to look at things a bit closer when I get back from summit.